### PR TITLE
Several minor enhancements and fixes

### PR DESCRIPTION
**Pull request by rpelisse**

### CODE REVIEW

**Comment by rpelisse** [comment]
@sabre1041 FYI, I need the last commit (adding minimum tasks/main.yml) to be merged for the tutorial.

File: tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
```
Comment (sabre1041) [inline tasks/main.yml:1]: @rpelisse this hardcodes it to a linux machine. I would add a condition to ensure you are running on a linux machine

**Review by sabre1041 — CHANGES_REQUESTED** [review]
@rpelisse Added 1 comment

**Comment by rpelisse** [comment]
@sabre1041 can we disable (or limit to warning) the yamlint stuff ? right now this PR is blocked because some license file is missing. I don't want to add a license file without some serious discussion about it first. And I don't have a lot of time to work on this before Monday, this kind of sidetracking me right now...

**Comment by sabre1041** [comment]
Will discuss license at a later date

**Pull request merged**